design/quarterly-release.md

```diff
@@ -0,0 +1,49 @@
+# Change Contour releases to quarterly
+
+Status: Accepted
+
+## Abstract
+Move Contour and Contour Operator to a longer release cadence to cushion for release planning, align Contour releases with Contour Operator releases, and align with downstream VMware TKG releases, and improve feature planning. 
+
+Please see [#3634](https://github.com/projectcontour/contour/issues/3634) for more discussion.
+
+## Current issues with monthly release
+* Very tough for users to keep up with Contour monthly releases. It’s rare for production deployments to always chase the latest version and more likely to upgrade every 6 months or longer. As part of these changes, we will be investigating how possible enabling jump upgrades is for Contour.
+* Release engineering is taxing and we have to spend some time preparing for this every month. The tradeoff here is that we will need to do more patch releases, and that quarterly releases will take more effort to put together than monthly ones, simply because they'll be bigger.
```

Comment:
Another huge tradeoff is back-porting features to two different releases. That was one of the reasons we started with monthly to avoid having to deal with patching backwards.

Reply:
I added a separate section for this. Is that better?